code/modules/mob/living/silicon/robot/robot_items.dm

```diff
@@ -171,7 +171,7 @@
 		else if(T.dead) //It's probably dead otherwise.
 			T.remove_dead(user)
 	else
-		to_chat(user, "Harvesting \a [target] is not the purpose of this tool.  The [src] is for plants being grown.")
+		to_chat(user, "Harvesting \a [target] is not the purpose of this tool.  The harvester is for plants being grown.")
```

Comment:
This is a Polaris line, so this would probably have to go upstream.

Also, it would be better to just remove The and keep [src]

Reply:
Derp, right. I forgot that, yeah.